### PR TITLE
Fixed melee bug, some typos.

### PR DESCRIPTION
**Pull request by Risusama**

### CODE REVIEW

File: DayZ/init/players/login_init.lua [hunk 1]
```diff
@@ -304,19 +304,19 @@ function playerLogin(username, pass, player)
 	local weapon = playerStatusTable[player]["currentweapon_1"] 
 	if weapon then
 		local ammoData,weapID = getWeaponAmmoFromName (weapon)
-			if (ammoData == "others") then
-				giveWeapon(player,weapID,1, true )
-			else 
+		if (ammoData == "others") then
+			giveWeapon(player,weapID,1, true )
+		else
```
Comment (neves768) [inline DayZ/init/players/login_init.lua:309]: Thanks for fixing it. +1

File: DayZ/gui/gear/language_gear.lua [hunk 2]
```diff
@@ -75,7 +75,7 @@ languageTextTable = {
 			{"M9",5, 'm9sd.png', 128, 128,"Handgun", "Ammunition: 9x19mm Cartridge","M9","Equip Secondary Weapon"},
 			{"Makarov SD",5,'g17.png',128,128,"Handgun","Ammunition: 9x19mm Cartridge","Makarov SD","Equip Secondary Weapon"},
 			{"PDW",5, 'pdw.png', 128, 128,"Sub-Machine Gun", "Ammunition: 9x19mm Cartridge","PDW","Equip Secondary Weapon"},
-			{"MP5A5",5, 'mp5a5.png', 128, 128,"Sub-Machine Gun", "Ammunition: 9x18mm Cartridge","MP5A5","Equip Secondary Weapon"},
+			{"MP5A5",5, 'mp5a5.png', 128, 128,"Sub-Machine Gun", "Ammunition: 9x19mm Cartridge","MP5A5","Equip Secondary Weapon"},
```
Comment (neves768) [inline DayZ/gui/gear/language_gear.lua:78]: Great fix. I don't know why It was 9x18mm before.
https://web.archive.org/web/20130514104452/http://www.hk-usa.com/military_products/mp5sd_specs.asp

File: DayZ/handlers/players/server/damage_player.lua [hunk 3]
```diff
@@ -241,21 +241,33 @@ function rearmPlayerWeapon (weaponName,slot)
...
 	local weapon = playerStatusTable[source]["currentweapon_1"]
 	if weapon then
 		local ammoData,weapID = getWeaponAmmoFromName (weapon)
-		giveWeapon(source,weapID,getElementData(source,ammoData), true )
+		if (ammoData == "others") then
+				giveWeapon(source,weapID,1, true )
+		else 
+			local ammoData,weapID = getWeaponAmmoFromName (weapon)
+			giveWeapon(source,weapID,getElementData(source,ammoData), true )
+		end
```
Comment (neves768) [inline DayZ/handlers/players/server/damage_player.lua]: I was going to comment about indentation, but you've done a great work here. Is it tested?

**Review by neves768 — COMMENTED** [review]
Do you tested it?

File: DayZ/handlers/players/server/damage_player.lua (hunk 3 above)
Comment (Risusama) [reply DayZ/handlers/players/server/damage_player.lua]: Yeah, I tested it for 20minutes before I made the commit, then notified people in the discord server that they may want to test it, and it seemed to work for them too. Also did a much longer test run yesterday, played with a friend for like 3 hours. Works like a charm, although equipping the hatchet is still a bit weird. It works, but it's weird. Probably something to do with it being the only melee weapon to use the Carrying/Holding-functionality.

Comment (Risusama) [reply DayZ/handlers/players/server/damage_player.lua]: I have a server running at the moment, so I can give you the address on discord if you'd like to test it too :)

Comment (neves768) [reply DayZ/handlers/players/server/damage_player.lua]: I'm gonna join it soon ;)

**Review by neves768 — APPROVED** [review]
Thanks again for your contribution ^^

**Pull request merged**